src/components/checkboxinput.js

```diff
@@ -30,7 +30,7 @@
     const parsedLabel = useText(label);
     const labelText = parsedLabel;
     const resolvedValue = useText(value);
-    const [checked, setChecked] = useState(boolify(resolvedValue));
+    const [checked, setChecked] = usePageState(boolify(resolvedValue));
```

Comment:
I think usePageState probably still has to be added to the linter file in order for the pipelines to pass here

Reply:
missing name has been added to linter-settings.